main.go

```diff
@@ -1,9 +1,16 @@
 package main
 
 import (
+	"fmt"
+
 	"github.com/France-ioi/AlgoreaBackend/cmd"
 )
 
+var version = "unknown"
```

Comment:
It might be cleaner to write
```
var (
   version = "unknown"
  _ = version
)
```
here instead of adding a dummy code into the main function

Reply:
fixed